yb-voyager/cmd/importData.go

```diff
@@ -1292,6 +1363,52 @@ func classifyTasks(state *ImportDataState, tasks []*ImportFileTask) (pendingTask
 	return append(inProgressTasks, notStartedTasks...), completedTasks, nil
 }
 
+func getNotStartedTasks(state *ImportDataState, tasks []*ImportFileTask) []*ImportFileTask {
```

Comment:
Can we instead club all this logic with `classifyTasks` since it's already doing very similar stuff?
Currently, it returns `pendingTasks, completedTasks`
What if we modify it to return `inProgressTasks,notStartedTasks,completedTasks` 

That should solve it right? We can avoid having separate functions. 
What do you think?

Reply:
i think its fine to have separate getter function. In classifyTasks we can reuse these.

Reply:
I still think it's better to merge this logic in classifyTasks. 
classifyTasks does not have to re-use these methods, because it is iterating through all the tasks and then classifying them. 

If you are keeping these methods, pls fix them to return proper errors, instead of utils.ErrExits